src/main/java/ai/elimu/model/enums/analytics/TaskType.java

```diff
@@ -5,8 +5,10 @@
 import java.util.Set;
 import ai.elimu.model.enums.content.LiteracySkill;
 import ai.elimu.model.enums.content.NumeracySkill;
+import lombok.AllArgsConstructor;
 
 // TODO: rename to "LearningType" or "ActivityType"?
+@AllArgsConstructor
 public enum TaskType {
```

Comment:
I'm getting an error message in the editor (NetBeans): 

![screen shot 2017-11-14 at 00 03 10](https://user-images.githubusercontent.com/1451036/32753975-59c8c528-c8cf-11e7-8a54-afae9a3a520d.png)

Reply:
I guess you need to tune your NetBeans to recognise Lombok. Try this? https://projectlombok.org/setup/netbeans

Reply:
Is it good now? Did you try this plugin for NetBeans?